addons/l10n_it_edi/models/account_move.py

```diff
@@ -463,7 +461,7 @@ def _l10n_it_edi_is_simplified(self):
         return bool(
             template_reference
             and not self.l10n_it_edi_is_self_invoice
-            and self._l10n_it_edi_export_buyer_data_check()
+            and list(buyer._l10n_it_edi_export_check().keys()) == ['partner_address_missing']
```

Comment:
Yes, if there are more errors then we couldn't use a simplified invoice instead of a normal one either: simplified can only be used if only the address is missing and otherwise the partner would be ok.